client/my-sites/domains/domain-management/edit/gdpr-consent-status-messages.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { gdprConsentStatus } from 'lib/domains/constants';
+
+function getGdprConsentStatusMessage( domain ) {
+	let message;
+
+	switch ( domain.gdprConsentStatus ) {
+		case gdprConsentStatus.NONE:
+			message = translate( 'Not available' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Not Available' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).